README.md

```diff
@@ -879,33 +887,43 @@ def a(self, x=0):
 Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds/all-contributors#emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+
```

Comment:
This section is autogenerated by the all-contributors tool, can we undo the modification?

Reply:
I didn't touch these lines. Not sure whats the issue. Can you check again and let me know what needs to be done. I just edited the summary lines now as there were line breaks.